webapp/channels/src/components/copy_button.tsx

```diff
@@ -22,12 +22,18 @@ type Props = {
 
 const CopyButton: React.FC<Props> = (props: Props) => {
     const [isCopied, setIsCopied] = useState(false);
+    const timerRef = useRef<NodeJS.Timeout | null>(null);
 
     const copyText = (e: React.MouseEvent<HTMLAnchorElement, MouseEvent>): void => {
         e.preventDefault();
         setIsCopied(true);
 
-        setTimeout(() => {
+        if (timerRef.current) {
+            clearTimeout(timerRef.current);
+            timerRef.current = null;
```

Comment:
we probably dont need to assign it to null because we are assigning it in the next line

Reply:
yes that sounds good, Ive done the change